### PR TITLE
Cosmetic improvements on figures

### LINKED ISSUE #35: Cosmetic improvements on figures
Todo:
- Add `ManufacturersModelName` embedded in each bar
- Add within-vendor stats as dashed line (mean) and transparent rectangle (std), and display mean+-std (cov)
- Add small flag of country for each site

See below a prototype (made with Keynote):
Screen Shot 2019-05-05 at 14 57 15

### PR DESCRIPTION
**Pull request by charleygros**
### Todo:

- [x] Add `ManufacturersModelName` embedded in each bar
- [x] Add within-vendor stats as dashed line (mean) and transparent rectangle (std), and display mean+-std (cov)
- [x] Add small flag of country for each site
- [x] Only show the top part of the error bar

Fixes #35

### CODE REVIEW

File: processing/generate_figure.py [hunk 1]
```diff
@@ -175,6 +175,19 @@ def aggregate_per_site(dict_results, metric, path_data):
     return results_agg
 
 
+def label_bar_model(bar_plot, model_lst):
```
Comment (jcohenadad) [inline processing/generate_figure.py]: thanks for adding an external function-- it is much cleaner 👍

**Comment by charleygros** [comment]
Add ManufacturersModelName embedded in each bar

**Comment by jcohenadad** [comment]
> Add ManufacturersModelName embedded in each bar

beautiful! could you please use bold font?

**Comment by charleygros** [comment]
Only show the top part of the error bar:

**Comment by charleygros** [comment]
I have not found an easy way to add emoji to matplotlib (as mentioned in #35).
--> I am suggesting a solution involving a folder with flag images.
Images are all from: https://emojipedia.org/ (eg https://emojipedia.org/flag-for-switzerland/).

Herebelow the result:

**Comment by charleygros** [comment]
Add within-vendor stats as dashed line (mean) and transparent rectangle (std), and display mean+-std (cov):

**Comment by charleygros** [comment]
@jcohenadad : Alignment of VendorModel + Missing flags + Justice for Switzerland --> corrected!
Thank you for your feedback :)

**Comment by jcohenadad** [comment]
Thank you so much @charleygros !!!!

**Pull request merged**